### ISSUE TITLE
observing priorities not initialized

### ISSUE THREAD

**Issue by forero** [desihub/desitarget#183]
After reading the targetmask [yaml file](https://github.com/desihub/desitarget/blob/master/py/desitarget/data/targetmask.yaml) the [target mask initialization](https://github.com/desihub/desitarget/blob/master/py/desitarget/targetmask.py#L51)  sets
the priorities for targets 'DONE', 'OBS' and 'DONOTOBSERVE' to `1`:

```python
In [1]: from desitarget import desi_mask, bgs_mask, mws_mask

In [2]: desi_mask['ELG'].priorities
Out[2]: 
{'DONE': 1,
 'DONOTOBSERVE': 1,
 'MORE_ZGOOD': 3000,
 'MORE_ZWARN': 3000,
 'OBS': 1,
 'UNOBS': 3000}

In [3]: desi_mask['LRG'].priorities
Out[3]: 
{'DONE': 1,
 'DONOTOBSERVE': 1,
 'MORE_ZGOOD': 3200,
 'MORE_ZWARN': 3200,
 'OBS': 1,
 'UNOBS': 3200}

In [4]: desi_mask['QSO'].priorities
Out[4]: 
{'DONE': 1,
 'DONOTOBSERVE': 1,
 'MORE_ZGOOD': 3500,
 'MORE_ZWARN': 3400,
 'OBS': 1,
 'UNOBS': 3400}
```
We should explicitly set those priorities to other values different from `1`. What should we do?

**Comment by sbailey** [comment]
Thanks for tracking this down.  It will somewhat clutter the yaml file, but it probably would be best to be explicit about these defaults.  Let's use:
```
DONOTOBSERVE: 0
OBS: 1
DONE: 2
```
That keeps all those states at low priority while still distinguishing them and allowing low-priority reobservation of observed targets that haven't been flagged as DONOTOBSERVE.

**Comment by moustakas** [comment]
Fixed in #184.
